src/api/forms/repositories/form-metadata-repository.js

```diff
@@ -28,30 +28,113 @@ export async function listAll() {
 }
 
 /**
- * Retrieves the list of documents from the database with pagination.
- * @param {PaginationOptions} options
+ * Retrieves the list of documents from the database with pagination and sorting.
+ * @param {QueryOptions} options - Pagination and sorting options
  * @returns {Promise<{ documents: WithId<Partial<FormMetadataDocument>>[], totalItems: number }>}
  */
-export async function list({ page = 1, perPage = MAX_RESULTS }) {
-  const coll = /** @type {Collection<Partial<FormMetadataDocument>>} */ (
-    db.collection(METADATA_COLLECTION_NAME)
-  )
+export async function list(options) {
+  try {
+    const {
+      page = 1,
+      perPage = MAX_RESULTS,
+      sortBy = 'updatedAt',
+      order = 'desc'
+    } = options
+
+    const coll =
+      /** @type {Collection<WithId<Partial<FormMetadataDocument>>>} */ (
+        db.collection(METADATA_COLLECTION_NAME)
+      )
+
+    const skip = (page - 1) * perPage
+
+    const pipeline = []
+
+    /** @type {CollationOptions | null} */
+    let collation = null
+
+    switch (sortBy) {
+      case 'updatedAt':
+        pipeline.push(
+          {
+            $addFields: {
+              updatedDateOnly: {
```

Comment:
I'd pull these into a var so you can re-use them for the default pipeline. Saves them drifting in the future.